research-hub-web/src/app/app.component.service.ts

```diff
@@ -4,8 +4,8 @@ import { Subject, Observable, BehaviorSubject } from 'rxjs';
 @Injectable()
 export class AppComponentService {
 
+  public url: Subject<string> = new Subject<string>();
   public titleChange: Subject<string> = new Subject<string>();
-  public customCSSClassName: Subject<string> = new Subject<string>();
```

Comment:
This obviously breaks `/researchimpact` which is what was using it (as it had its own unique layout/CSS). But that's fine, we don't need it anymore.